nitro-protocol/contracts/examples/HashLockedSwap.sol

```diff
@@ -17,11 +17,10 @@ contract HashLockedSwap is IForceMoveApp {
     function validTransition(
         VariablePart memory a,
         VariablePart memory b,
-        uint48 turnNumB,
         uint256
     ) public override pure returns (bool) {
         // is this the first and only swap?
-        require(turnNumB == 4, 'turnNumB != 4');
+        require(b.turnNum == 4, 'turnNumB != 4');
```

Comment:
Nit: 
```suggestion
        require(b.turnNum == 4, 'b.turnNum != 4');
```

Reply:
Implemented in [`894e94d`](https://github.com/statechannels/go-nitro/pull/508/commits/894e94db7ea9d87ae2ce9118feb144561abe920c)